library.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Monocypher",
+    "description": "An easy to use, easy to deploy crypto library. It provides functions for authenticated encryption, hashing, password key derivation, key exchange, and public key signatures.",
+    "keywords":  "encryption,signing,key-exchange,password-key-derivation",
+    "version": "2.0.8",
```

Comment:
Monocypher's latest version is "2.0.6". Typo?

Also, the tarball generator scripts automatically scans for "__git__", and replaces that with the actual version. If I were to accept that pull request, we'll need to write

        "version": "__git__",

Reply:
ouch, indeed, 2.0.6 :)

platformio works in a way that this file is retreived from the VCS, and then it points to either the local directory, or a specific GH repo, and then retreives the release taged with `2.0.6` using the GH api. So this file is like a fixed pointer to the last release, I agree, a bit annoying, but if you remove the version field, it publishes all commits, not just the tags you pick.